### DISCUSSION TITLE
Anyone figured out Color Picker For Tabs?

### DISCUSSION THREAD

**Discussion by SlavicBlin** [tupoy-ya/TupoyeMenu#100] [Q&A] [answered]
In the menu GUI settings for button colors it does not change the tab button colors.
I know in the code it states thats its on the ( // TODO: Use GUI Color. ) list but i was wondering if anyone has figured out how to do it yet as the colors in the menu could use some sparkle, i have manually changed the color to be a fixed Grey color rather than the blue but i have never got it to work with the color picker in the settings as i am very unexperinced when it comes to doing something like that especially with ImVec4 and ImU32.

Any suggestions / ways to get it working would be nice.

**Comment by tupoy-ya** [comment, marked as answer]
See if b170d992a69b82379bfe10666110788184d5c26b fixes this.

**Reply by SlavicBlin** [reply]
alright ill look into it in about an hour, thanks in advance <3

**Reply by SlavicBlin** [reply]
Yeap works great thanks
